BendAssist.App/BendAssists/MakeFlange.cs

```diff
@@ -14,46 +14,47 @@ public sealed class MakeFlange : BendAssist {
    public MakeFlange (Part part) => mPart = part;
    #endregion
 
+   #region Properties -----------------------------------------------
+   public override string[] Prompts => ["Select an edge to make flange [Ctrl= All edges]"];
+   #endregion
+
    #region Methods --------------------------------------------------
    public override bool Assisted () { return mCanAssist; }
 
    /// <summary>Adds a new flange with the specified height, angle and radius.</summary>
    /// Translates the selected pline to the given height
-   /// and creates a bend line with the given angle.
+   /// Creates a bend line with the given angle.
    public override void Execute () {
       if (mPart == null) return;
       int index = 1;
       List<PLine> pLines = []; List<BendLine> bendLines = [];
       foreach (var bendline in mPart.BendLines) bendLines?.Add (bendline);
       foreach (var pline in mPart.PLines) {
          var (startPt, endPt) = (pline.StartPoint, pline.EndPoint);
-
          // Radially moves the point perpendicular to the selected pline
          var angle = CommonUtils.ToRadians (pline.Angle - 90);
          var bendDeduction = BendUtils.GetBendDeduction (mBendAngle, 0.38, 2, mRadius);
-         mHeight -= bendDeduction / 2; // Gets the bend deducted height of the flange
-
+         var height = mHeight;
+         height -= bendDeduction / 2; // Gets the bend deducted height of the flange
          // Calculates the offsets in x and y
-         var (dx, dy) = (mHeight * Math.Cos (angle), mHeight * Math.Sin (angle));
-
+         var (dx, dy) = (height * Math.Cos (angle), height * Math.Sin (angle));
          // Translates the line with the offset values
          var translatedLine = (PLine)pline.Translated (dx, dy);
-
-         pLines?.Add (new PLine (pline.StartPoint, translatedLine.StartPoint, index++));
-         pLines?.Add (new PLine (translatedLine.StartPoint, translatedLine.EndPoint, index++));
-         pLines?.Add (new PLine (translatedLine.EndPoint, pline.EndPoint, index++));
-
+         pLines?.Add (new (pline.StartPoint, translatedLine.StartPoint, index++));

```

Comment:
var (tStart, tEnd) = (translatedLine.StartPoint, translatedLine.EndPoint) and reuse them in the code.